Update README (#8)
BetterThanTomorrow/calva-backseat-driver: README.md

    "backseat-driver": {
      "command": "node",
      "args": [

>         "<absolute path to calva-mcp-server.js in the extension folder>",

Reviewer: Yes, that's very much clearer!

        "<absolute path to port file (which points to your project's .calva/mcp-server/port)"
      ]
    }
  }
}
```

> Both absolute paths needed above can be conveniently determined by clicking on the **Copy command** button (shown when starting the MCP server) and then pasting into `mcp.json` file, als described in section [Configuration](#configuration-if-using-mcp-server) above.

Reviewer: ```suggestion
Both absolute paths needed above can be conveniently determined by clicking on the **Copy command** button (shown when starting the MCP server) and then pasting into `mcp.json` file, as described above: [Configuration](#configuration-if-using-mcp-server).
```

    "backseat-driver": {
      "command": "node",
      "args": [

>         "<absolute path to wrapper script calva-mcp-server.js>",

Author: I had the assumtion that Claude Code does not support the VS Code extension, therefore different wording. I might be wrong, haven't used Claude Code, yet.

>         "<absolute path to wrapper script calva-mcp-server.js>",

Author: Fixed
